superset-frontend/plugins/plugin-chart-echarts/src/Histogram/buildQuery.ts

```diff
@@ -25,8 +25,10 @@ export default function buildQuery(formData: HistogramFormData) {
   return buildQueryContext(formData, baseQueryObject => [
     {
       ...baseQueryObject,
+      extras: { where: `${column} IS NOT NULL` },
       columns: [...groupby, column],
       post_processing: [histogramOperator(formData, baseQueryObject)],
+      metrics: undefined,
```

Comment:
`metrics` by default is `[]` but we need it to be `undefined` to [avoid a GROUP BY](https://github.com/apache/superset/blob/a9cebe81ea8cd1ae56df05d5807baa8f05398cc5/superset/models/helpers.py#L1525).